cirq-google/cirq_google/engine/engine.py

```diff
@@ -834,7 +835,7 @@ def get_engine(project_id: Optional[str] = None) -> Engine:
 
     Args:
         project_id: If set overrides the project id obtained from the
-            environment variable `GOOGLE_CLOUD_PROJECT`.
```

Comment:
Thanks for that; I'm happy to go through and clean up more of those usages. To be clear, `google.auth.default()` does seem to respect the `GOOGLE_CLOUD_PROJECT` env var, so it's fine to set the project via the environment _or_ with `gcloud set project`. It'd be nice if we can include some boilerplate language about this in one location and reference that elsewhere (at least for code; notebooks should probably be self-contained). Do you have a preference about where to document this?

Reply:
SGTM. It may be worth mentioning this in the "concepts" page [here](https://quantumai.google/cirq/google/concepts#cloud_project), but it looks like it may have been avoided to lean on the "getting started" tutorial to discuss creation.

It looks like validation only happens in one place today and that the other cases are only in documentation and tests, so maybe we should just generalize the docstrings and point them to the shared documentation (referenced above?) about what it means to "set a project". WDYT?

I don't want to spiral your change out of control, so approving now if you'd like to capture this in a followup issue instead.